assets/styles.css

```diff
@@ -1,7 +1,5 @@
-@import url("/assets/norwester.otf") format("opentype");
```

Comment:
The first loading error I saw when I viewed my fork in gitpitch.com was from this line. I removed it in the hopes that it would fix all of my loading problems, but it did not. It does appear that the slideshow is using default sans-serif to render anyway, so I just removed this font.